lib/help.js

```diff
@@ -141,10 +141,10 @@ class Help {
    */
 
   subcommandTerm(cmd) {
-    // Legacy. Ignores custom usage string, and nested commands.
-    const args = cmd._args.map(arg => humanReadableArgName(arg)).join(' ');
-    return cmd._name +
-      (cmd._aliases[0] ? '|' + cmd._aliases[0] : '') +
+    const term = cmd._name + (cmd._aliases[0] ? '|' + cmd._aliases[0] : '');
```

Comment:
Minor: "term" is used to describe the whole thing, so I would like a different variable name here. Maybe cmdNames?